src/Layout/Navigation/MainList.tsx

```diff
@@ -36,7 +29,7 @@ function NavigationMainList(props: { activePath: string }): React.ReactElement {
         item={item}
         activePath={props.activePath}
       >
-        {item.icon}
+        {item.icon && (item.icon)}
```

Comment:
Why don't you remove this line?

Reply:
I thought about the way to return all icons back. I can remove it.